### ISSUE TITLE
Altdorf game crash

### ISSUE THREAD

**Issue by thudir** [WarEmu/WarBugs#8794]
Whenever I go to Altdorf after visiting another area I have a crash game

**Comment by Luth-RoR** [comment]
Are there any  _details_ you can provide? Like something useful?

**Comment by Soulwhisper** [comment]
To me happens mostly when i use Scroll of Altdorf.
By the time i spawn in Market after 5-6 i move it crashes.

**Comment by thudir** [comment]
I do not do any action? I just pass in the northern part of altdorf: usually on the market place.

**Comment by sesabu** [comment]
mee to, when I come altdorf by flymaster I ALLWAIS have a game crash in the arrival platform or steping down the stairs. Then when I enter the game again I popup near the market and no problem

**Comment by Soulwhisper** [comment]
When this happened to me many times i crashed in Altdorf and i couldnt connect in game with my character(Which is already known). I should type fast 2times /stuck then if i used Scroll of Altdorf and went back in market after 5-6 seconds i crashed.

**Comment by Luth-RoR** [comment]
Strange issue; recently for me it happened only once after porting to altdorf and not every time after i visited another area.

**Comment by Luth-RoR** [comment]
Does this still happen?

**Comment by Blaqwar** [comment]
It happens on and off for me, some days I can log in and immediatelly crash from Altdorf or just crash after porting but most of the time it's fine. I've learnt to deal with it because as much as I've tried to pinpoint a cause I haven't been able (my attempts at resolving this have also been dismissed by you, Luth, so I gave up :D).

I should add that it only happens in Altdorf and that my connection is incredibly stable (I have one connection drop from my side per 5-6 weeks).
